bin/stacks/routing-api-stack.ts

```diff
@@ -259,22 +261,123 @@ export class RoutingAPIStack extends cdk.Stack {
         period: Duration.minutes(5),
         statistic: 'p90',
       }),
-      threshold: 7500,
+      threshold: 5500,
```

Comment:
Would this make the alarm more sensitive?

Reply:
that's right